libraries/Adapters/Microsoft.Bot.Builder.Adapters.Twilio/TwilioAdapter.cs

```diff
@@ -131,9 +128,18 @@ public async Task ProcessAsync(HttpRequest request, HttpResponse response, IBot
                         Id = twilioEvent.MessageSid,
                         Timestamp = new DateTime(),
                         ChannelId = "twilio-sms",
-                        Conversation = new ConversationAccount() { Id = twilioEvent.From, },
-                        From = new ChannelAccount() { Id = twilioEvent.From, },
-                        Recipient = new ChannelAccount() { Id = twilioEvent.To, },
+                        Conversation = new ConversationAccount()
```

Comment:
did resharper ask for this change in format? I was never quite sure why it was changed to a single line in the first place